### ISSUE TITLE
Nirspec WCS pipeline should use fix_inputs

### ISSUE THREAD

**Issue by stscijgbot** [spacetelescope/jwst#4083]
Issue [JP-1028](https://jira.stsci.edu/browse/JP-1028) was created by Nadia Dencheva:

Depends on AL-187

**Comment by stscijgbot** [comment]
Comment by Nadia Dencheva: Putting this on hold. It's work that won't be completed for build 7.9. It is also not necessary for the pipeline to work, it's just a nice improvement in terms of user interface.

The FS mode was converted in [https://github.com/nden/jwst/tree/nrs-noncoord-inputs. |https://github.com/nden/jwst/tree/nrs-noncoord-inputs.]

This was used as a proof that the concept and implementation in gwcs work as expected.

No work has been done on IFU mode.

**Comment by melanieclarke** [comment]
Work is complete, with #9404.  See JP ticket for more information.
